articles/scopes/current/index.md

```diff
@@ -13,12 +13,12 @@ useCase:
 ---
 # Scopes
 
-The OAuth 2.0 protocol is a delegated authorization mechanism, where an application requests access to resources controlled by the user (the resource owner) and hosted by an API (the resource server), and the authorization server issues the application a more restricted set of credentials than those of the user.
+The OAuth 2.0 protocol is a delegated authorization mechanism, where an application requests access to resources controlled by the user (the resource owner) and hosted by an API (the resource server). The authorization server issues the application a more restricted set of credentials than those of the user in the form of an Access Token.
 
-The **scope** parameter allows the application to express the desired scope of the access request. The **scope** parameter can also be used by the authorization server in the response to indicate which scopes were actually granted (if they are different than the ones requested).
+The permissions represented by the Access Token in OAuth 2.0 terms are known as scopes. The `scope` parameter allows the application to express the desired scope of the access request. The `scope` parameter can also be used by the authorization server in the response to indicate which scopes were actually granted (if they are different than the ones requested).
 
 You can use scopes to:
 
-- Let an application authenticate users and get additional information about them, such as their email or picture. For details, refer to [OpenID Connect Scopes](/scopes/current/oidc-scopes).
+- Indicate that the application wants to verify the identity of the user (by using [Open ID Connect](/protocols/oidc)), as well as to obtain basic profile information about the user, such as their email or picture. For details, refer to [OpenID Connect Scopes](/scopes/current/oidc-scopes).
```

Comment:
Audience shift? This article is written to a user who owns the API (the first bullet point was and the second bullet point still is).